### ISSUE TITLE
Some settings cannot be changed

### ISSUE THREAD

**Issue by svepe** [slgobinath/SafeEyes#187]
First of all, thanks for the great app - it is simple, with a lot of useful functionality and to the point! However, I experience some issues with it and I will be happy to help investigating and fixing them.

I am running on Ubuntu 14.04 with cinnamon and cannot change the settings 'Allow postponing the breaks' and 'Audible alert at the end of break'. I can enable them in the settings dialog, save the changes and close the dialog, however if I reopen the dialog the settings have not changed. Furthermore, I do get an audio sound after the end of each break, but I am not able to postpone breaks.

**Comment by slgobinath** [comment]
Hi,
Could you please follow these steps and let me know the results?
1. Terminate safeeyes
2. Remove the safeeeyes config directory
```
rm -r ~/.config/safeeyes
```
3. Start safeeyes again
4. Make sure that  `~/.config/safeeyes` directory has READ & WRITE permissions
5. Change the settings
6. Restart safeeyes

If you still have the same problem, please share the `~/.config/safeeyes/safeeyes.json` file.

**Comment by svepe** [comment]
I have followed the steps your recommended and now 'Allow postponing the breaks' behaves normally, however I still cannot get the 'Audible alert at the end of break'. Right now it is enabled in the settings file, but according to the GUI it is off. Here is the contents of my `safeeyes.json`:

```
{
    "active_window_class": {
        "skip_break": [],
        "take_break": []
    },
    "allow_postpone": true,
    "audible_alert": true,
    "break_interval": 15,
    "custom_exercises": {},
    "disable_options": [
        {
            "label": "for_x_minutes",
            "time": 30,
            "unit": "minute"
        },
        {
            "label": "for_x_hour",
            "time": 1,
            "unit": "hour"
        },
        {
            "label": "for_x_hours",
            "time": 2,
            "unit": "hour"
        },
        {
            "label": "for_x_hours",
            "time": 3,
            "unit": "hour"
        }
    ],
    "enable_screen_lock": true,
    "idle_time": 1,
    "language": "system",
    "lock_screen_command": [],
    "long_break_duration": 60,
    "long_breaks": [
        {
            "name": "long_break_walk"
        },
        {
            "name": "long_break_lean_back"
        }
    ],
    "meta": {
        "config_version": "5.0.1"
    },
    "no_of_short_breaks_per_long_break": 3,
    "persist_state": true,
    "plugins": [],
    "postpone_duration": 5,
    "pre_break_warning_time": 20,
    "short_break_duration": 15,
    "short_breaks": [
        {
            "name": "short_break_close_eyes"
        },
        {
            "name": "short_break_roll_eyes"
        },
        {
            "name": "short_break_rotate_clockwise"
        },
        {
            "name": "short_break_rotate_counter_clockwise"
        },
        {
            "name": "short_break_blink"
        },
        {
            "name": "short_break_focus_far_distance"
        },
        {
            "name": "short_break_drink_water"
        }
    ],
    "shortcut_disable_time": 2,
    "shortcut_postpone": 65,
    "shortcut_skip": 9,
    "show_time_in_tray": false,
    "strict_break": false,
    "time_to_screen_lock": 5
}
```

**Comment by slgobinath** [comment]
Hi,
I have tested it with my Ubuntu 14.04 VM and bug is confirmed. There is something wrong with `pyaudio` I will release the fix asap.

Thanks for your effort and support :-)

**Comment by slgobinath** [comment]
The error is due to `pyaudio`:
```
ALSA lib pcm.c:7843:(snd_pcm_recover) underrun occurred
```

**Comment by svepe** [comment]
From what I gathered underrun occurs when data is not sent to the audio stream buffer at a sufficiently high rate. I can see that you are reading from the .wav file and sending to the audio stream in the same [`while` loop](https://github.com/slgobinath/SafeEyes/blob/589f101de421111512857cc85ea47f2cd7983a14/safeeyes/Utility.py#L76) which might be the reason for the underrun. Maybe you would want to substitute the `while` with a `for` (for efficiency reasons) and do the reading and writing in separate loops as shown [here](https://www.programcreek.com/python/example/52624/pyaudio.PyAudio).

If this does not work, you could try something like [this](https://stackoverflow.com/questions/19230983/prevent-alsa-underruns-with-pyaudio)? Also the problem may be caused by the audio file.

**Comment by slgobinath** [comment]
I think `aplay` will take the problems away from us. Because installing `pyaudio` using `pip` has some issues (at least in Ubuntu and Debian) and already an issue regarding `pyaudio` was reported. See https://github.com/slgobinath/SafeEyes/issues/135.

**Comment by slgobinath** [comment]
I have implemented the [Audible Alert](https://github.com/slgobinath/SafeEyes/blob/safeeyes-2.0.0/safeeyes/plugins/audiblealert/plugin.py#L50) plugin with `aplay` in Safe Eyes 2.0.0. Since I invest my time on 2.0.0(which will be the next release), I am not going to fix this issue in 1.2.x series.
